sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -138,9 +141,17 @@ class SQLContext(@transient val sparkContext: SparkContext)
   @transient
   protected[sql] lazy val catalog: Catalog = new SimpleCatalog(conf)
 
+  protected[sql] val extendedFunctions: Map[String, FunctionBuilder] = Map(
```

Comment:
can you just inline this into the functionRegistry call? I want to expose as few functions as possible.